testing/test_pdb.py

```diff
@@ -481,14 +480,15 @@ def test_supports_breakpoint_module_global(self):
             assert SUPPORTS_BREAKPOINT_BUILTIN is False
 
     @pytest.mark.skipif(not SUPPORTS_BREAKPOINT_BUILTIN, reason="Requires breakpoint() builtin")
-    def test_sys_breakpointhook_not_custom_pdb(self):
+    def test_sys_breakpointhook_is_pytest_PDB(self):
         """
-        Test that sys.breakpointhook is not set to the custom Pdb class
+        Test that sys.breakpointhook by default is set to pytest's internal class.
         """
-        assert sys.breakpointhook != pytestPDB.set_trace
+        assert sys.breakpointhook == pytestPDB.set_trace
```

Comment:
This test seemed incorrect to me and it was failing on my system: wasn't `sys.breakpoint` supposed to be set in the normal circumstance? I changed the logic and docstring to reflect that and it passes. @tonybaloney please double check here.